app/views/foods/search_results.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render partial: '/partials/search_nav_bar' %>
+
+<h3>Total Results: <%= @total_hits %></h3>
+
+<h5>Top Ten Results: </h5>
+
+<% if @top_ten_foods.length > 0 %>
```

Comment:
Love the use of the conditional here to account for searches that return nothing!